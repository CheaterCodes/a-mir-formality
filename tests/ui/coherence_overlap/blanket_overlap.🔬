[
    crate core {
        trait Foo { }

        trait Bar { }

        impl<ty T> Foo for T { }
        
        impl<ty T> Foo for T where T: Bar { }
    }
]