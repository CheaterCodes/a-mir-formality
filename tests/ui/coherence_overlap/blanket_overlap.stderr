Error: impls may overlap:
impl <ty> Foo for ^ty0_0 { }
impl <ty> Foo for ^ty0_0 where ^ty0_0 : Bar { }
